src/docs/product/sentry-basics/search/searchable-properties/session-replay.mdx

```diff
@@ -120,6 +120,12 @@ The number of URLs that the user visited during a replay recording.
 
 - **Type:** number
 
+### `dead.selector`
+
+Similar to the `click.selector` search property, but only queries on dead clicks. An element identified using a subset of CSS selector syntax. For example, `#section-1` or `span.active` or `span[role=button]` or `.active[role=button]` would all match the element `<span id="section-1" class="active" role="button"/>`. Note that, CSS combinators, pseudo selectors, and attr selectors other than `=` are not supported.
```

Comment:
Should we define or link the `dead clicks` in the first sentence?

Reply:
Added a link to where we define dead/rage clicks in the docs already

Reply:
woo! link right to the text too!